superset-frontend/src/explore/components/ControlPanelsContainer.jsx

```diff
@@ -76,13 +76,6 @@ class ControlPanelsContainer extends React.Component {
       // apply current value in formData
       value: formData[name],
     };
-    const { mapStateToProps: mapFn } = controlData;
-    if (mapFn) {
-      Object.assign(
-        controlData,
-        mapFn(exploreState, controlData, actions) || {},
-      );
-    }
```

Comment:
No need to call `mapStateToProps` again because it's always called during both [initialization](https://github.com/apache/incubator-superset/blob/e0ccb2ce1d592e6b259c6c95831ecff502a10224/superset-frontend/src/explore/controlUtils.js#L205) and [reduce actions](https://github.com/apache/incubator-superset/blob/194f9ef9be82d9532bd55b34a35e68dfc22e9bc3/superset-frontend/src/explore/reducers/exploreReducer.js#L100) (via the `getControlStateFromControlConfig` function).